app/views/bars/show.html.erb

```diff
@@ -37,14 +40,24 @@
     </div>
   </div>
 
-<div>
   <% @reviews.order(created_at: :desc).each do |review| %>
-    <div><%= review.title + " " + review.rating.to_s + " " + review.body %></div>
+    <div class="review">
+      <h3><%= review.title %></h3>
+      <h4><%= review.rating %>/5</h4>
+      <p><%= review.body %></p>
```

Comment:
You should be using `<div>` tags instead of `<p>` tags here as you'll probably be styling this somehow later on. No need to change this now, but keep it in mind when you flesh all this out.